scipy/stats/_continuous_distns.py

```diff
@@ -947,13 +987,11 @@ class expon_gen(rv_continuous):
     -----
     The probability density function for `expon` is::
 
-        expon.pdf(x) = lambda * exp(- lambda*x)
+        expon.pdf(x) = exp(-x)
 
     for ``x >= 0``.
 
-    The scale parameter is equal to ``scale = 1.0 / lambda``.
-
-    `expon` does not have shape parameters.
```

Comment:
Does this still mention that the rate parameter  `lambda =  1 / scale` for the common parameterization, see #4538